keyboards/pikatea/pikatea_macropad_gb5/pikatea_macropad_gb5.c

```diff
@@ -16,27 +16,6 @@
 
 #include "pikatea_macropad_gb5.h"
 
-#if defined(ENCODER_ENABLE)
-bool encoder_update_kb(uint8_t index, bool clockwise) {
```

Comment:
@justinknguyen can you explain why this would be removed? doesn't the other keymaps need this?

Reply:
it works without it, we just need this
![image](https://github.com/PikateaCompany/vial-qmk/assets/20637584/1a980404-4e08-4f48-aaa5-a90b70c8c063)